assets/reducers/popup.js

```diff
@@ -9,7 +10,8 @@ const defaultState = {
     point: null,
     isFetchingElevation: false,
     elevation: null,
-    values: []
+    values: [],
+    region: ''
```

Comment:
I usually flag everything as required by default, then change it as necessary (like in this case).